pkg/cluster/cluster.go

```diff
@@ -333,6 +333,7 @@ func (c *Cluster) startSeedMember(recoverFromBackup bool) error {
 	m := &etcdutil.Member{
 		Name:      etcdutil.CreateMemberName(c.cluster.Metadata.Name, c.memberCounter),
 		Namespace: c.cluster.Metadata.Namespace,
+		Secure:    c.isSecure(),
```

Comment:
The plan is to implement client TLS later. There is no need to differentiate the two here.

I think once it "isSecure()", we will have this member have both TLS.